mbs_results/ratio_of_means.py

```diff
@@ -332,14 +339,24 @@ def ratio_of_means(
     if filters is not None:
 
         df = flag_rows_to_ignore(df, filters)
+    
```

Comment:
Just so I am clear with the understanding, you have added this if statement to check if we are in a case with pre-constructed links. Otherwise we do the other method which Anton wrote previously?

Reply:
Yes. If the user specifies the pre-calculated link columns with the new parameter, then the first couple of steps in the pipeline which calculate the imputation links are skipped. Then it carries on the rest of the pipeline as usual.